lib/taskana-core/src/test/java/acceptance/task/DeleteTaskAccTest.java

```diff
@@ -0,0 +1,139 @@
+import pro.taskana.exceptions.InvalidStateException;
+import pro.taskana.exceptions.InvalidWorkbasketException;
+import pro.taskana.exceptions.NotAuthorizedException;
+import pro.taskana.exceptions.TaskAlreadyExistException;
+import pro.taskana.exceptions.TaskNotFoundException;
+import pro.taskana.exceptions.WorkbasketNotFoundException;
+import pro.taskana.security.JAASRunner;
+import pro.taskana.security.WithAccessId;
+
+/**
+ * Acceptance test for all "delete task" scenarios.
+ */
+@RunWith(JAASRunner.class)
+public class DeleteTaskAccTest extends AbstractAccTest {
+
+    public DeleteTaskAccTest() {
+        super();
+    }
+
+    @Ignore
+    @WithAccessId(
+        userName = "user_1_2",
+        groupNames = {"group_1"})
+    @Test(expected = TaskNotFoundException.class)
+    public void testDeleteSingleTask()
+        throws SQLException, NotAuthorizedException, InvalidArgumentException, ClassificationNotFoundException,
+        WorkbasketNotFoundException, TaskAlreadyExistException, InvalidWorkbasketException, TaskNotFoundException,
+        ConcurrencyException, AttachmentPersistenceException {
+
+        TaskService taskService = taskanaEngine.getTaskService();
+        Task task = taskService.getTask("TKI:000000000000000000000000000000000036");
+
+        // taskService.deleteTask(task.getId());
+
+        Task deletedTask = taskService.getTask("TKI:000000000000000000000000000000000036");
+    }
+
+    @Ignore
+    @WithAccessId(
+        userName = "user_1_2",
+        groupNames = {"group_1"})
+    @Test(expected = InvalidStateException.class)
+    public void testThrowsExceptionIfTaskIsNotCompleted()
+        throws SQLException, NotAuthorizedException, InvalidArgumentException, ClassificationNotFoundException,
+        WorkbasketNotFoundException, TaskAlreadyExistException, InvalidWorkbasketException, TaskNotFoundException,
+        ConcurrencyException, AttachmentPersistenceException {
+
+        TaskService taskService = taskanaEngine.getTaskService();
+        Task task = taskService.getTask("TKI:000000000000000000000000000000000027");
+
+        // taskService.deleteTask(task.getId());
+    }
+
+    @Ignore
+    @WithAccessId(
+        userName = "user_1_2",
+        groupNames = {"group_1"})
+    @Test(expected = TaskNotFoundException.class)
+    public void testForceDeleteTaskIfNotCompleted()
+        throws SQLException, NotAuthorizedException, InvalidArgumentException, ClassificationNotFoundException,
+        WorkbasketNotFoundException, TaskAlreadyExistException, InvalidWorkbasketException, TaskNotFoundException,
+        ConcurrencyException, AttachmentPersistenceException {
+
+        TaskService taskService = taskanaEngine.getTaskService();
+        Task task = taskService.getTask("TKI:000000000000000000000000000000000027");
+
+        // taskService.deleteTask(task.getId(), true);
+
+        Task deletedTask = taskService.getTask("TKI:000000000000000000000000000000000036");
```

Comment:
getTask should probably get task wit Id "TKI:000000000000000000000000000000000027"